### ISSUE TITLE
Afficher les dataset_name (observations et fiches espèces)

### ISSUE THREAD

**Issue by AudreyEnGuyane** [PnX-SI/GeoNature-atlas#418]
Bonjour à tous,

J'aimerai ajouter les sources de données dans l'atlas:
1. dans les fenêtres donnant le détail des observations (sur clic) le jeu de données source.
J'imagine que les données proviennent de la vm_observations (donc y ajouter le champ dataset_name) mais quid du paramétrage pour l'ajouter à l'affichage?

2. ajouter un onglet "Sources de données" dans les fiches espèces, à côté de ceux des communes et observateurs. Quelles sont les sources de données de ces deux onglets? Comment en ajouter un 3e?

Merci d'avance!

**Comment by camillemonchicourt** [comment]
Dans la version 1.5.0 de GeoNature-atlas on a ajouté les organismes sources des données dans les fiches de chaque espèces, mais aussi en leur créant une fiche par organisme : https://github.com/PnX-SI/GeoNature-atlas/issues/291

Cela permet de valoriser les producteurs de données.
N'est-ce pas plutôt cela que tu souhaites faire ?

Pour l'affichage des jeux de données, à voir si c'est vraiment une information lisible et compréhensible pour le public ?
Mais en effet ça pourrait être une info complémentaire. 
A voir comment faire ça correctement et le rendre activable ou non grâce à un paramètre (comme les organismes).

**Comment by AudreyEnGuyane** [comment]
Merci Camille. C'est en effet, une belle première étape pour rendre à César ce qui lui appartient.

Par contre, attention! Les fiches "organismes" affichent toutes les données de la base, et non pas seulement celles qui sont diffusées. C'est embêtant car bon nombre de nos données sont intégrées pour consultation/exploitation en interne mais pas pour diffusion.

Je n'ai pas trouvé la source de la requête sur laquelle se base cette fiche, ce qui me permettrait d'ajouter un filtre sur les données compilées. Peux-tu m'éclairer?

**Comment by AudreyEnGuyane** [comment]
_Je ne vois pas comment il pourrait remonter des données qui ne sont pas dans GeoNature-atlas ?!?_
Pour exemple, j'ai bien 6 données visibles mais les références des organismes me liste l'ensemble des données, issues de la vm_observations (donc les 36 données d'une asso + celles d'un autre organisme tiers...).

En fait, j'ai fini par trouver: les fiches organisme sont basées sur vm_cor_taxon_organisme qui traite l'ensemble des données, quelque soit le niveau de diffusion. En modifiant la source de la vm en ajoutant un filtre diffusion_level <> 140, c'est tout bon.
Pour info si d'autres en ont besoin:

```
    CREATE MATERIALIZED VIEW atlas.vm_cor_taxon_organism AS
 SELECT sum(obs_by_dataset_and_orga.nb_obs) AS nb_observations,
    obs_by_dataset_and_orga.cd_ref,
    bo.id_organisme AS id_organism,
    bo.nom_organisme AS nom_organism,
    bo.adresse_organisme AS adresse_organism,
    bo.cp_organisme AS cp_organism,
    bo.ville_organisme AS ville_organism,
    bo.tel_organisme AS tel_organism,
    bo.email_organisme AS email_organism,
    bo.url_organisme AS url_organism,
    bo.url_logo
   FROM ( SELECT rcda.id_dataset,
            count(obs.id_observation) AS nb_obs,
            obs.cd_ref,
            rcda.id_organism
           FROM atlas.vm_observations obs
             JOIN utilisateurs.reduced_cor_dataset_actor rcda ON obs.id_dataset = rcda.id_dataset
          WHERE obs.diffusion_level <> 140
          GROUP BY rcda.id_dataset, obs.cd_ref, rcda.id_organism) obs_by_dataset_and_orga
     JOIN utilisateurs.bib_organismes bo ON bo.id_organisme = obs_by_dataset_and_orga.id_organism
  GROUP BY obs_by_dataset_and_orga.cd_ref, bo.id_organisme, bo.nom_organisme, bo.adresse_organisme, bo.cp_organisme, bo.ville_organisme, bo.tel_organisme, bo.email_organisme, bo.url_organisme, bo.url_logo;
```

**Comment by camillemonchicourt** [comment]
OK donc il y a un soucis à corriger au niveau de cette VM si je comprends bien ?
Dans ce cas, il vaut mieux laisser le ticket ouvert.

**Comment by AudreyEnGuyane** [comment]
Oui, en effet, il faudra le corriger. Pour une fois que j'apporte la solution :-D ! 

A noter que pour soulager le traitement des données de l'tlas, ça peut même être fait au niveau de la vm_observations. Je ne l'ai pas fait à ce niveau car la modifier impliquait de supprimer/restaurer bon nombre d'autres vm qui en dépendait.
